employee-rostering-backend/README.adoc

```diff
@@ -1 +1,57 @@
-// TODO Add instructions for running the application and backend
\ No newline at end of file
+= OptaWeb Employee Rostering Backend
+
+image:https://sonarcloud.io/api/project_badges/measure?project=employee-rostering&metric=alert_status[
+"Quality Gate Status", link="https://sonarcloud.io/dashboard?id=optaweb-employee-rostering"]
+image:https://sonarcloud.io/api/project_badges/measure?project=optaweb-employee-rostering&metric=reliability_rating[
+"Reliability Rating", link="https://sonarcloud.io/dashboard?id=optaweb-employee-rostering"]
+image:https://sonarcloud.io/api/project_badges/measure?project=optaweb-employee-rostering&metric=security_rating[
+"Security Rating", link="https://sonarcloud.io/dashboard?id=optaweb-employee-rostering"]
+image:https://sonarcloud.io/api/project_badges/measure?project=optaweb-employee-rostering&metric=sqale_rating[
+"Maintainability Rating", link="https://sonarcloud.io/dashboard?id=optaweb-employee-rostering"]
+image:https://sonarcloud.io/api/project_badges/measure?project=optaweb-employee-rostering&metric=ncloc[
+"Lines of Code", link="https://sonarcloud.io/dashboard?id=optaweb-employee-rostering"]
+image:https://sonarcloud.io/api/project_badges/measure?project=optaweb-employee-rostering&metric=coverage[
+"Coverage", link="https://sonarcloud.io/dashboard?id=optaweb-employee-rostering"]
+
+image:https://david-dm.org/kiegroup/optaweb-employee-rostering/status.svg?path=optaweb-employee-rostering[
+"dependencies Status", link="https://david-dm.org/kiegroup/optaweb-employee-rostering?path=optaweb-employee-rostering"]
```

Comment:
What is david-dm.org? Is it a reliable service (= do we expect it to keep running in 3 years from now - 90% confidence or more) @rsynek Do we use david-dm.org elsewhere?

Reply:
Never heard of. Interestingly, the link gives me "Internal Server Error". Is there anything that must be done to on the service first?

Reply:
IIRC, @yurloc used these links in vehicle routing. @yurloc, thoughts?

Reply:
my bad, it's in the readme of a "frontend" module of VRP.

Reply:
My mistake, david-dm.org shows `node.js` dependencies and does not belong in the backend documentation. I will remove it.